_data/changelogs/component-date-range-picker.yml

```diff
@@ -2,6 +2,13 @@ title: Date range picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added Date range picker usability guidance.
+    summaryAdditional: Users should always be able to enter dates manually in Date range picker.
```

Comment:
**suggestion:** Let's be more specific about the type of guidance added. Just want to make it easy to immediately understand what was changed. Here is a possible suggestion: 
```suggestion
    summary: Added guidance about always allowing users to type in the date manually.
```

Note: We should update both the date picker and date range picker changelogs with the same summary.